src/GithubWebhookJira/Webhook.php

```diff
@@ -0,0 +1,276 @@
+<?php
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017 NocWorx
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ */
+
+namespace GithubWebhookJira;
```

Comment:
should be in the `NocWorx` namespace?

Reply:
It was originally written outside of NocWorx, so I didn't do that. But if thats preferred I can redo it.

Reply:
This is a good point in general.  As we transition from "NocWorx" team to "Software Development" what should we do namespace-wise?  Is it a matter of in which GitRepo it resides (nexcess vs. nocworx vs thermo)?

Reply:
Personally I think Nexcess would be a better namespace here, and for most things that are not NocWorx itself.

Reply:
Y, I tend to agree. @robertology ? WHAT SAY YOU?

Reply:
:+1: